internal/awsservice/dynamodb.go

```diff
@@ -68,7 +68,7 @@ func GetItemInDatabase(databaseName, indexName string, checkingAttribute, checki
 			"#second_attribute": checkingAttribute[1],
 		},
 		ExpressionAttributeValues: map[string]types.AttributeValue{
-			":first_attribute":  &types.AttributeValueMemberN{Value: checkingAttributeValue[0]},
+			":first_attribute":  &types.AttributeValueMemberS{Value: checkingAttributeValue[0]},
```

Comment:
Just to confirm if this is because of the type change in attribute in `setup/main.tf`

Reply:
Yes that's correct. Since the S is for String and N is for Number and for upcoming PR, I will search based on CommitHash and CWA Use Case which both are String.